ObsFacilityWikidata.tex

```diff
@@ -126,7 +129,8 @@ \section{Observation Facility Semantic Scope}
 shown in Figure \ref{fig:pds4-concept-map}, and defined as follows:
 ``\emph{The Observing System class describes the entire suite used to 
 collect the data}''. The list of \texttt{Observing\_System\_Components} 
-is defined as: \emph{Airborne, Aircraft, Artificial Illumination, 
+is\todo{Umm -- this is not mentioned in the figure?}
```

Comment:
Indeed, this is mentioned somewhere else in the information model. I should be more precise here. The list of terms is available here: https://pds.nasa.gov/datastandards/documents/im/v1/index_1H00.html#attribute_pds_observing_system_component_pds_type

Reply:
This is fixed in b893444